src/components/timeline/TimelinePanel.svelte

```diff
@@ -161,6 +164,32 @@
     }
   }
 
+  function getSelectedActivityDirective(): ActivityDirective | undefined {
+    if ($selectedActivityDirectiveId !== null) {
+      return $activityDirectivesMap[$selectedActivityDirectiveId];
+    }
+  }
+
+  function onKeyDownOverPanel({ detail }: CustomEvent<KeyboardEvent>) {
```

Comment:
I think my concern with this approach would be that it is a little less predictable as to when exactly I can paste. For example, if i click on a directive in the table and then hover over the timeline _without clicking on it_ i can't paste the directive. Curious to hear pros and cons of this vs always allowing directive paste _except_ when the event comes from an input?

Reply:
Also not sure if this is the root cause but something in this PR has broken date picker day selection. But this feels like something that could be involved somehow?